src/testdir/test_vim9_import.vim

```diff
@@ -1140,29 +1140,34 @@ def Test_autoload_import_relative()
   v9.CheckScriptFailure(lines, 'E484:')
 enddef
 
-def Test_autoload_import_relative_compiled()
-  # autoload relative, access from compiled function. #14565
-  var lines =<< trim END
-    vim9script
+def Test_autoload_import_relative_compiled_buffer()
+  if !has('unix')
+    CheckUnix
```

Comment:
return?

Reply:
I copied the pattern. The entire test is in the else. I'll change it to early return and get rid of the else. (My preference anyway)

Reply:
@chrisbra DONE. Early return if not unix.

Reply:
thanks!